set up rn v13.7.0 (#2036)
mia-platform/documentation: docs/release-notes/v13.7.0.md

---
id: v13.7.0
title: Version 13.7.0 Release Notes
sidebar_label: v13.7.0
image: "img/release-note-link-preview.png"
---

_March 20th, 2025_

:::info

Mia-Platform Console v13.7.0 is **now in Preview** and will be generally available on April 03rd.

Console SaaS users can try out v13.7.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.

For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
:::

## Monitor Audit Logs of your Company

TO DO

## Console

### Improvements

### Bug Fix

This version address the following bugs:

* we have fixed a bug that prevented the correct visualization of the "Documentation" tab in the Endpoints settings when using API Portal v2;
* we fixed a bug in which the securityInfo of an endpoint was not added when a single subroute of an endpoint was exposed.

## Fast Data

> ### Fast Data Services
> 
> ####
> 
> ##### Fixed

Reviewer: ```suggestion
### Fast Data Services

#### Single View Creator

The new versions `v6.8.1` of the _Single View Creator - MongoDB_ and _Single View Creator - Kafka_ are available!

##### Fixed
- When performing aggregation, the service now correctly handles ER-schemas that contain a condition with a `NULL` equality

For more information check out the service [changelog](/runtime_suite/single-view-creator/changelog.md).
```

---
id: v13.7.0
title: Version 13.7.0 Release Notes
sidebar_label: v13.7.0
image: "img/release-note-link-preview.png"
---

_March 20th, 2025_

:::info

Mia-Platform Console v13.7.0 is **now in Preview** and will be generally available on April 03rd.

Console SaaS users can try out v13.7.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.

For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
:::

## Monitor Audit Logs of your Company

The new **Audit Logs** feature is now available, providing a powerful tool to effortlessly monitor all activities within your company!  

> For more details, check out the [Audit Logs documentation](/).

Reviewer 2: this should be updated with the correct link, the doc has been merged in main

To enable this feature in self-hosted installations check out dedicated section in the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md#upgrade-from-v13.6.2-to-v13.7.0)

## Console

### Improvements

### Bug Fix

This version address the following bugs:

* we have fixed a bug that prevented the correct visualization of the "Documentation" tab in the Endpoints settings when using API Portal v2;
* we fixed a bug in which the securityInfo of an endpoint was not added when a single subroute of an endpoint was exposed.

## Fast Data

### Fast Data Services

####

##### Fixed

## Marketplace

### Marketplace Updates

> ####

Reviewer: ```suggestion
#### CRUD Service

The new version `7.2.3` of the CRUD Service is available!

### Changed

* overhauled Mongo view update logic to avoid requesting [`dropCollection`](https://www.mongodb.com/docs/manual/reference/privilege-actions/#mongodb-authaction-dropCollection) privilege action
* introduce abort controller to NodeJS stream pipeline to ensure the pipeline object is cleaned up when the response stream is destroyed

### Fixed

* prevent a memory leak the service was experiencing when it was not able to fulfill incoming requests
* corrected a typo in option `maxIdleTimeMS` within Mongo client configuration, which prevented the environment variable `MONGODB_MAX_IDLE_TIME_MS` to take effect
```

## How to update your Console

> For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `x.x.x`.

Reviewer 2: ```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `14.0.0-beta.0`.

```

@ThisIsDemetrio FYI

The new **Audit Logs** feature is now available, providing a powerful tool to effortlessly monitor all activities within your Company!

![Audit logs table](./img/audit-logs.png)

For more details, check out the [Audit Logs documentation](/development_suite/identity-and-access-management/monitor-audit-logs.md).

To enable this feature in self-hosted installations check out the dedicated section in the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md#upgrade-from-v1362-to-v1370)

## Console

### Bug Fix

This version address the following bugs:

* A bug causing the graphical display of the Runtime section for Infrastructure Resources to malfunction has been resolved;
* We have resolved a bug that prevented the correct display of the 'Documentation' tab in Endpoints settings when using API Portal v2;
* We resolved a bug where the securityInfo of an endpoint was not added when a single subroute of that endpoint was exposed.

## Fast Data

### Fast Data Services

#### Single View Creator

The new versions `v6.8.1` of the _Single View Creator - MongoDB_ and _Single View Creator - Kafka_ are available!

> ##### Fixed

Reviewer: ```suggestion
##### Fixed

```

- When performing aggregation, the service now correctly handles ER-schemas that contain a condition with a `NULL` equality

For more information check out the service [changelog](/runtime_suite/single-view-creator/changelog.md).

## Marketplace

### Marketplace Updates

#### CRUD Service

The new version `7.2.3` of the CRUD Service is available!

### Changed

* overhauled Mongo view update logic to avoid requesting [`dropCollection`](https://www.mongodb.com/docs/manual/reference/privilege-actions/#mongodb-authaction-dropCollection) privilege action
* introduce abort controller to NodeJS stream pipeline to ensure the pipeline object is cleaned up when the response stream is destroyed

> ### Fixed

Reviewer: ```suggestion
##### Fixed

```
